website/docs/d/glue_catalog_table.html.markdown

```diff
@@ -0,0 +1,116 @@
+
+## Attributes Reference
+
+* `id` - Catalog ID, Database name and of the name table.
+* `arn` - The ARN of the Glue Table.
+* `description` - Description of the table.
+* `owner` - Owner of the table.
+* `parameters` - Properties associated with this table, as a list of key-value pairs.
+* `partition_index` - Configuration block for a maximum of 3 partition indexes. See [`partition_index`](#partition_index) below.
+* `partition_keys` - Configuration block of columns by which the table is partitioned. Only primitive types are supported as partition keys. See [`partition_keys`](#partition_keys) below.
+* `retention` - Retention time for this table.
+* `storage_descriptor` - Configuration block for information about the physical storage of this table. For more information, refer to the [Glue Developer Guide](https://docs.aws.amazon.com/glue/latest/dg/aws-glue-api-catalog-tables.html#aws-glue-api-catalog-tables-StorageDescriptor). See [`storage_descriptor`](#storage_descriptor) below.
+* `table_type` - Type of this table (EXTERNAL_TABLE, VIRTUAL_VIEW, etc.). While optional, some Athena DDL queries such as `ALTER TABLE` and `SHOW CREATE TABLE` will fail if this argument is empty.
+* `target_table` - Configuration block of a target table for resource linking. See [`target_table`](#target_table) below.
+* `view_expanded_text` - If the table is a view, the expanded text of the view; otherwise null.
+* `view_original_text` - If the table is a view, the original text of the view; otherwise null.
+
+### partition_index
+
+* `index_name` - Name of the partition index.
+* `keys` - Keys for the partition index.
+
+### partition_keys
+
+* `comment` - Free-form text comment.
+* `name` - Name of the Partition Key.
+* `type` - Datatype of data in the Partition Key.
+
+### storage_descriptor
+
+* `bucket_columns` - List of reducer grouping columns, clustering columns, and bucketing columns in the table.
+* `columns` - Configuration block for columns in the table. See [`columns`](#columns) below.
+* `compressed` - Whether the data in the table is compressed.
+* `input_format` - Input format: SequenceFileInputFormat (binary), or TextInputFormat, or a custom format.
+* `location` - Physical location of the table. By default, this takes the form of the warehouse location, followed by the database location in the warehouse, followed by the table name.
+* `number_of_buckets` - Is if the table contains any dimension columns.
+* `output_format` - Output format: SequenceFileOutputFormat (binary), or IgnoreKeyTextOutputFormat, or a custom format.
+* `parameters` - User-supplied properties in key-value form.
+* `schema_reference` - Object that references a schema stored in the AWS Glue Schema Registry. See [Schema Reference](#schema-reference) below.
+* `ser_de_info` - Configuration block for serialization and deserialization ("SerDe") information. See [`ser_de_info`](#ser_de_info) below.
```

Comment:
Lets create some consistency between the paragraph references, I noticed diferent ways being used.

>  See [`ser_de_info`](#ser_de_info) below
> See [Schema Reference](#schema-reference) below